lib/vc.js

```diff
@@ -120,14 +117,19 @@ async function issue(options = {}) {
     throw new TypeError('"suite.verificationMethod" property is required.');
   }
 
-  // run common credential checks
-  const {credential} = options;
   if(!credential) {
     throw new TypeError('"credential" parameter is required for issuing.');
   }
+
+  // Set the issuance date to now(), if missing
+  const now = (new Date()).toJSON();
+  credential.issuanceDate = credential.issuanceDate ||
+    `${now.substr(0, now.length - 5)}Z`;
+
+  // run common credential checks
   _checkCredential(credential);
 
-  return jsigs.sign(credential, {purpose, documentLoader, suite, ...options});
+  return jsigs.sign(credential, {purpose, documentLoader, suite, expansionMap});
```

Comment:
I don't know if this is ok -- were there other options that may have been passed through?

Reply:
@dlongley so that's part of the reason why I figured it's ok to drop the general `...options` -- we got rid of all the other variables to pass through, so might as well list them explicitly).